resources/v1/01_users.sql

```diff
@@ -0,0 +1,13 @@
+    CREATE TABLE users
+    (
+       id               INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+       first_name       VARCHAR (32) NOT NULL,
+       last_name        VARCHAR (32) NOT NULL,
+       email            VARCHAR (32) NOT NULL UNIQUE,
+       mobile_no        VARCHAR (10) NOT NULL UNIQUE,
+       otp              VARCHAR (4),
+       otp_generated_at DATETIME, 
+       current_location POINT,
+       created_at       DATETIME DEFAULT CURRENT_TIMESTAMP NOT NULL 
+
+    )
```

Comment:
we might need ); in the end , not sure though varies from version to version, 
in my mysql version, it throws error, does it works in your version?

Reply:
Yes it is working in my version.